### PR TITLE
Feature/delete event

### PR DESCRIPTION
**Pull request by aussie-m-mike**
https://trello.com/c/m44Pv8kU/371-delete-event-endpoint
https://trello.com/c/DFD1LUXq/372-delete-event-from-google-calendar
https://trello.com/c/NzaEuvbX/375-store-events-ids-from-calendars

### CODE REVIEW

File: api/app/Repositories/Event/EventRepository.php [hunk 1]
```diff
@@ -63,4 +63,14 @@ public function getEventsEmails(
...
     }
+
+    public function getById(int $id): ?Event
+    {
+        return Event::find($id);
+    }
+
+    public function deleteById(int $id): void
+    {
+        Event::destroy($id);
```
Comment (lenchvolodymyr) [inline api/app/Repositories/Event/EventRepository.php:74]: I thinnk, we should cancel events in application, not delete

**Review by lenchvolodymyr — CHANGES_REQUESTED** [review]

File: api/app/Repositories/Event/EventRepository.php (hunk 1 above)
Comment (aussie-m-mike) [reply api/app/Repositories/Event/EventRepository.php:74]: Change to cancelled status + added reschedule ability

**Comment by lenchvolodymyr** [comment]
While you resolve conflicts, could you also add criteria to not query canceled events when a reminder is sent?
https://github.com/BinaryStudioAcademy/bsa-2020-schedulia/blob/develop/api/app/Jobs/SendNotificationBeforeEvent.php#L39

**Comment by aussie-m-mike** [comment]
Criterion has been added & conflicts has been merged

**Review by lenchvolodymyr — APPROVED** [review]

**Pull request merged**